Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/type/enums/TypeInterface.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.data.elasticsearch.index.type.enums;
+
+import javax.json.JsonArray;
+import javax.json.JsonObject;
+
+import org.kitodo.data.exceptions.DataException;
+
+interface TypeInterface {
+
+    /**
+     * Get boolean value from given json object.
+     * 
+     * @param jsonObject
+     *            returned from ElasticSearch index
+     * @return boolean value for given json
+     */
+    default boolean getBooleanValue(JsonObject jsonObject) throws DataException {
+        try {
+            return jsonObject.getBoolean(this.toString());
+        } catch (RuntimeException e) {
+            throw new DataException("Not possible to retrieve boolean value for key " + this.toString()
+                    + ". Exception: " + e.getMessage());
+        }
+    }
+
+    /**
+     * Get int value from given json object.
+     * 
+     * @param jsonObject
+     *            returned from ElasticSearch index
+     * @return int value for given json
+     */
+    default int getIntValue(JsonObject jsonObject) throws DataException {
+        try {
+            return jsonObject.getInt(this.toString());
+        } catch (RuntimeException e) {
+            throw new DataException("Not possible to retrieve int value for key " + this.toString()
+                    + ". Exception: " + e.getMessage());
+        }
+    }
+
+    /**
+     * Get String value from given json object.
+     * 
+     * @param jsonObject
+     *            returned from ElasticSearch index
+     * @return String value for given json
+     */
+    default String getStringValue(JsonObject jsonObject) throws DataException {
+        try {
+            return jsonObject.getString(this.toString());
+        } catch (RuntimeException e) {
+            throw new DataException("Not possible to retrieve String value for key " + this.toString()
+                    + ". Exception: " + e.getMessage());
+        }
+    }
+
+    /**
+     * Get json array value from given json object.
+     *
+     * @param jsonObject
+     *            returned from ElasticSearch index
+     * @return json array for given json
+     */
+    default JsonArray getJsonArray(JsonObject jsonObject) throws DataException {
+        try {
+            return jsonObject.getJsonArray(this.toString());
+        } catch (RuntimeException e) {
```

Comment:
Why did you catch any `RuntimeException` based class here? So far as I can see only `ClassCastException` is official thrown.